libs/elements/layout/header/src/lib/components/header/header.component.scss

```diff
@@ -0,0 +1,147 @@
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:opsz,wght@9..40,400;9..40,700&family=Roboto:wght@100;300;400;500;700;900&display=swap');
```

Comment:
DMSans is locally set on the project just fetch the latest changes. But good job on composing your styles